rotkehlchen/api/v1/schemas.py

```diff
@@ -1074,6 +1074,16 @@ class BlockchainAccountDataSchema(Schema):
     label = fields.String(load_default=None)
     tags = fields.List(fields.String(), load_default=None)
 
+    @validates_schema
+    def validate_blockchain_account_schema(  # pylint: disable=no-self-use
+            self,
+            data: Dict[str, Any],
+            **_kwargs: Any,
+    ) -> None:
+        label = data.get('label', None)
+        if label is not None and label == '':
```

Comment:
```suggestion
        if label == '':
```

`None == '' -> False`